homeassistant/components/habitica/coordinator.py

```diff
@@ -20,6 +20,8 @@
 
 from .const import DOMAIN
 
+PARALLEL_UPDATES = 0
```

Comment:
I believe this only does anything on a platform.

Reply:
I wasn't sure about that, saw another PR also adding this to the coordinator. But I think you are right. How is it about the actions registered in `async_setup`?